.github/issue_template/03_mybuildisbroken.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 😵 My Build is Broken
+about: Use this template for helping figure out what's wrong with your build.
```

Comment:
Should we instead push them to discussions? Once we, you know, have that . . .

Reply:
Absolutely once we have it 🙂